apps/OboeTester/app/src/main/java/com/mobileer/oboetester/AudioStreamBase.java

```diff
@@ -92,6 +94,8 @@ String dump(int framesPerBurst) {
             }
             StringBuffer buffer = new StringBuffer();
 
+            buffer.append("time between callbacks = " + callbackTimeStr + "\n");
+
```

Comment:
The line from 110 with the xRun# is no longer being displayed on screen. You neeed to change "statusView" at bottom of "stream_config.xml" to have 5 lines instead of 4.

Reply:
Thanks for the heads up! Done in iteration 3